### PR TITLE
Hotjar and GA Integration

### PR DESCRIPTION
**Pull request by madnihamza1841**
**Description:** 
Add Hotjar and GA tracking codes in LMS to enable learner user tracking across site.

**JIRA:** 
https://edlyio.atlassian.net/browse/EDLY-3972

**Testing instructions:**

1. Check that the Hotjar is showing user data for green dev
2. Check that user data is shown on Google Analytics Green Dev Property

**Merge checklist:**

- [ ] All reviewers approved
- [ ] CI build is green
- [ ] Documentation updated (not only docstrings)
- [ ] Commits are (reasonably) squashed
- [ ] Translations are updated

**Post merge:**
- [ ] Delete working branch (if not needed anymore)

**Note:** Please add screenshots for any view able change.

### CODE REVIEW

File: lms/envs/aws.py [hunk 1]
```diff
@@ -744,6 +744,9 @@
 GOOGLE_ANALYTICS_LINKEDIN = AUTH_TOKENS.get('GOOGLE_ANALYTICS_LINKEDIN')
 GOOGLE_SITE_VERIFICATION_ID = ENV_TOKENS.get('GOOGLE_SITE_VERIFICATION_ID')
 
+# HOTJAR ANALYTICS
+HOTJAR_TRACKING_ID = ENV_TOKENS.get('HOTJAR_TRACKING_ID')
```
Comment (burhandodhy) [inline lms/envs/aws.py:748]: No need to add it in this file

File: openedx/features/edly/context_processor.py [hunk 2]
```diff
@@ -66,6 +66,8 @@ def edly_app_context(request):  # pylint: disable=unused-argument
             'zendesk_widget_url': marketing_link('ZENDESK-WIDGET'),
             'footer_url': marketing_link('FOOTER'),
             'gtm_id': configuration_helpers.get_value('GTM_ID'),
+            'google_analytics_id': configuration_helpers.get_value('GA_ID'),
```
Comment (burhandodhy) [inline openedx/features/edly/context_processor.py:69]: For the consistency set `google_analytics_id` to `ga_id`

File: lms/envs/test.py [hunk 3]
```diff
@@ -594,3 +594,6 @@
 ############### Settings for Django Rate limit #####################
 
 RATELIMIT_RATE = '2/m'
+
+# HOTJAR ANALYTICS
+HOTJAR_TRACKING_ID = None
```
Comment (burhandodhy) [inline lms/envs/test.py:599]: I dont think we need it here.

File: lms/envs/aws.py (hunk 1 above)
Comment (madnihamza1841) [reply lms/envs/aws.py:748]: Sure. WIll remove it.

File: lms/envs/test.py (hunk 3 above)
Comment (madnihamza1841) [reply lms/envs/test.py:599]: Not adding it here breaks the code and starts to give errors. Debugged this last time with @taimoor-ahmed-1

**Review by taimoor-ahmed-1 — APPROVED** [review]

**Review by burhandodhy — APPROVED** [review]

**Pull request merged**